modules/consensus/consensusset.go

```diff
@@ -61,6 +61,10 @@ type ConsensusSet struct {
 	// marshaler encodes and decodes between objects and byte slices.
 	marshaler encoding.GenericMarshaler
```

Comment:
I'm going to merge this because it looks good, I'll try and find a good example with regards to the data structure formatting